ee/server/service/teams.go

```diff
@@ -496,6 +516,9 @@ func (svc *Service) ApplyTeamSpecs(ctx context.Context, specs []*fleet.TeamSpec,
 	}
 
 	if len(details) > 0 {
+		// TODO(mna): we don't create separate activities for e.g. edited agent
+		// options when applying team specs, so I didn't create explicit activities
+		// for min macos version either. Not sure if that's what we want.
```

Comment:
I pinged Noah here just to make sure: https://github.com/fleetdm/fleet/issues/9355#issuecomment-1411100489